uPortal-portlets/src/main/java/org/apereo/portal/portlets/search/portletregistry/PortletRegistrySearchService.java

```diff
@@ -17,128 +21,57 @@
 import java.util.List;
 import javax.portlet.PortletRequest;
 import javax.servlet.http.HttpServletRequest;
-import org.apereo.portal.portlet.PortletUtils;
 import org.apereo.portal.portlet.om.IPortletDefinition;
-import org.apereo.portal.portlet.om.IPortletWindow;
-import org.apereo.portal.portlet.om.IPortletWindowId;
 import org.apereo.portal.portlet.registry.IPortletDefinitionRegistry;
-import org.apereo.portal.portlet.registry.IPortletWindowRegistry;
-import org.apereo.portal.portlets.groupselector.EntityEnum;
 import org.apereo.portal.portlets.search.IPortalSearchService;
+import org.apereo.portal.rest.utils.PortletRegistryUtil;
 import org.apereo.portal.search.SearchRequest;
 import org.apereo.portal.search.SearchResult;
 import org.apereo.portal.search.SearchResults;
-import org.apereo.portal.security.IAuthorizationService;
 import org.apereo.portal.url.IPortalRequestUtils;
-import org.apereo.portal.url.IPortalUrlBuilder;
-import org.apereo.portal.url.IPortalUrlProvider;
-import org.apereo.portal.url.IPortletUrlBuilder;
-import org.apereo.portal.url.UrlType;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.beans.factory.annotation.Value;
 
+/**
+ * Return portlet search results based on title, name and description matching.
```

Comment:
Too specific. Needs a qualifier like "currently" or "such as". So that it's not an API change (SemVer!) when this enhances to match on additional factors or so.

Reply:
I will update the javadoc.